front-end/src/components/Post/GovernanceSideBar/VoteReferendum.tsx

```diff
@@ -41,6 +41,40 @@ const VoteRefrendum = ({ className, referendumId, address, accounts, onAccountCh
 		{ text: '5x time lock', value: 'Locked5x' }
 	];
 	const [conviction, setConviction] = useState<ConvictionType>(options[0].value);
+	const [totalVotes, setTotalVotes] = useState(0);
+	const [yesVotes, setYesVotes] = useState(0);
+	const [noVotes, setNoVotes] = useState(0);
+
+	useEffect(() => {
```

Comment:
Can we use a dedicated component rather than cluttering this `VoteReferendum`. This will explode one we add some more UI.

Reply:
done